index.html

```diff
@@ -69,7 +69,7 @@ <h2 class="section-title">New Activity</h2>
             <p class="alert-msg">Limit duration to 59 minutes and 59 seconds.</p>
           </div>
           <article class="start-container">
-            <button class="start-activity-btn">START ACTIVITY</button>
+            <button class="start-activity-btn btn">START ACTIVITY</button>
```

Comment:
It looks like you may have some styles that are shared between all of your buttons. I'd encourage you to pull those out into a the shared `btn` class and use it like you are here.